packages/logging/yarn.lock

```diff
@@ -2,33 +2,6 @@
 # yarn lockfile v1
 
 
-"@execonline-inc/environment@^1.0.2":
```

Comment:
This removal in this yarn.lock file appears to be a valid removal of items that shouldn't have been included, or were maybe once referenced in the past but lacked a yarn install since until now.